### ISSUE TITLE
Cannot read property 'getComputedStyle' of undefined

### ISSUE THREAD

**Issue by anibali** [react-grid-layout/react-draggable#186]
Version: react-draggable 2.2.0
Browser: Google Chrome Linux 51.0.2704.106 (64-bit)
Error: `Cannot read property 'getComputedStyle' of undefined`
Source of error: https://github.com/mzabriskie/react-draggable/blob/ac269435565fe584f0409345dcb79e8f5b682828/lib/utils/positionFns.es6#L29

Looking a few lines up from the source of the error, I think that changing

``` js
const ownerWindow = node.defaultView;
```

to

``` js
const ownerWindow = ownerDocument.defaultView;
```

might fix the problem. I've never used `defaultView` before though, and I'm not sure what is trying to be achieved with it, so take that with a pinch of salt.

I have no problems with react-draggable 2.1.2.

**Comment by grunnlock** [comment]
Happens to me as well when using the "bounds" property:
`<ReactDraggable
      bounds="body"
      handle=".dashboard-panel-header"
      onStart={ @focusPanel }
      onClick={ @focusPanel }
    >`

**Comment by afilp** [comment]
It happens to me too, this is very problematic and I do not know what to do! I would appreciate a quick fix, thanks a lot!

**Comment by CaptainChemist** [comment]
@anibali I ran into this same issue, did you end up fixing it? I too was able to fix it by reverting to an earlier version (2.1.2).

**Comment by CaptainChemist** [comment]
The weird thing with my project is that even if I revert back to a previously working version on github and do a `rm -rf node_modules` followed by an npm install, it still gives me this error. I wonder if this problem is something other than the project itself- it also appears to be broken in safari as well as chrome for me.

**Comment by anibali** [comment]
@CaptainChemist I ended up reverting to the earlier version. I did some poking around in the console and found that the window object was available via the document, but I didn't actually make the change. The line I'm referring to is https://github.com/mzabriskie/react-draggable/blob/ac269435565fe584f0409345dcb79e8f5b682828/lib/utils/positionFns.es6#L21

Essentially `node.defaultView` is `undefined`, and that's causing the problem.

**Comment by vhmth** [comment]
Confirmed that this is the case for me as well. Reverting to 2.1.2 indeed fixed the issue.

**Comment by jonshamir** [comment]
Any updates on this issue?

**Comment by uriklar** [comment]
+1

**Comment by STRML** [comment]
If you guys have an issue please submit a PR.

**Comment by STRML** [comment]
Published in v2.2.1. Thanks.
